src/Chapter1.hs

```diff
@@ -553,9 +556,10 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
-
-mid x y z = error "mid: not implemented!"
-
+mid :: (Num a, Ord a) => a -> a -> a -> a
+mid x y z = sum xyz - maximum xyz - minimum xyz 
+  where xyz = [x, y, z]
```

Comment:
That's a very clever implementation! :brain: 
Though, I wonder, will it still work around big numbers because of integer overflow :thinking: 
We don't test such cases in our course, but something to keep in mind when implementing it in real life :slightly_smiling_face:

Reply:
should've put Integer there 🙂